projectEuler/problem5.js

```diff
@@ -0,0 +1,28 @@
+// prompt: find the smallest positive number that is evenly divisible by all of the numbers from 1 to 20
+
+// helper function for checking if integer is divisible by all numbers from 1 to 20
+function isDivisible(int) {
+  var count = 0;
+  for (let i = 1; i <= 20; i++) {
+    if (int % i === 0) {
+      count++;
+    }
+  }
+  if (count === 20) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+// main function
+function smallestMult() {
```

Comment:
Generally prefer abbreviating most words in programming to enhance readability. Exceptions such as "calc", "str", "arr", "obj", etc exist. We can make this this function self-documenting by updating the name to include a clue as to the action we're performing. `calcSmallestMultiple` would do the trick. Adding `calc` may signal to other devs that we're doing a simple calculation here rather than performing a query, making an API request, or accessing the data from an object.

Reply:
@autumn-han sorry I had a typo in this comment. I meant "Generally prefer **not** abbreviating most words"